code/datums/brain_damage/special.dm

```diff
@@ -188,6 +188,56 @@
 	REMOVE_TRAIT(owner, TRAIT_SIXTHSENSE, TRAUMA_TRAIT)
 	active = FALSE
 
+/datum/brain_trauma/special/existential_crisis
+	name = "Existential Crisis"
+	desc = "Patient's hold on reality becomes faint, causing occasional bouts of non-existence."
+	scan_desc = "existential crisis"
+	gain_text = "<span class='notice'>You feel less real.</span>"
+	lose_text = "<span class='warning'>You feel more substantial again.</span>"
+	var/obj/effect/abstract/sync_holder/veil/veil
+	var/next_crisis = 0
+
+/datum/brain_trauma/special/existential_crisis/on_life()
+	..()
+	if(!veil && world.time > next_crisis && prob(3))
+		if(isturf(owner.loc))
+			fade_out()
+
+/datum/brain_trauma/special/existential_crisis/on_lose()
+	if(veil)
+		fade_in()
+	..()
+
+/datum/brain_trauma/special/existential_crisis/proc/fade_out()
+	if(veil)
+		return
+	var/duration = rand(50, 450)
+	veil = new(owner.drop_location())
```

Comment:
It may be a good idea to refactor sync_holders to also put users in stasis, now that stasis actually exists.

Reply:
This uses the same mechanic as the desynchronizer, which means the user can act as if they were in an invisible locker (swap equipment, drop items, radio communication, and so on). Given that, stasis is probably not warranted, although it makes sense for the immortality talisman version.